src/server.c

```diff
@@ -935,19 +935,19 @@ int clientsCronResizeOutputBuffer(client *c, mstime_t now_ms) {
  *
  * When we want to know what was recently the peak memory usage, we just scan
  * such few slots searching for the maximum value. */
-#define CLIENTS_PEAK_MEM_USAGE_SLOTS 8
 size_t ClientsPeakMemInput[CLIENTS_PEAK_MEM_USAGE_SLOTS] = {0};
 size_t ClientsPeakMemOutput[CLIENTS_PEAK_MEM_USAGE_SLOTS] = {0};
+int CURR_PEAK_MEM_USAGE_SLOT = 0;
```

Comment:
for gobal variable, let's use the following style

```suggestion
int CurrentPeakMemUsageSlot = 0
```

Reply:
Modifications completed, thanks~